internal/stream/event_stream_test.go

```diff
@@ -0,0 +1,191 @@
+package stream_test
+
+import (
+	"context"
+	"errors"
+	"io/ioutil"
+	"testing"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/golang/mock/gomock"
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/assisted-service/internal/stream"
+	"github.com/openshift/assisted-service/models"
+	"github.com/sirupsen/logrus"
+)
+
+func GetDummyEventStream(ctrl *gomock.Controller) *stream.MockEventStreamInterface {
```

Comment:
seems exactly the same as in ``test/dummies.go``, no?

Reply:
Oops, forgot to remove it from here :facepalm: